python/tests/res/fm/test_ecl_run_new.py

```diff
@@ -0,0 +1,197 @@
+#  Copyright (C) 2018  Equinor ASA, Norway.
+#
+#  The file 'test_ecl.py' is part of ERT - Ensemble based Reservoir Tool.
+#
+#  ERT is free software: you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation, either version 3 of the License, or
+#  (at your option) any later version.
+#
+#  ERT is distributed in the hope that it will be useful, but WITHOUT ANY
+#  WARRANTY; without even the implied warranty of MERCHANTABILITY or
+#  FITNESS FOR A PARTICULAR PURPOSE.
+#
+#  See the GNU General Public License at <http://www.gnu.org/licenses/gpl.html>
+#  for more details.
+import os
+import stat
+import unittest
+import shutil
+import functools
+import inspect
+import yaml
+import pytest
+import re
+
+from ecl.summary import EclSum
+from ecl.util.test import TestAreaContext
+from tests import ResTest
+from tests.utils import tmpdir
+from res.fm.ecl import *
```

Comment:
It would be nice to avoid these wildcard imports. The sudden `run` method further down comes from `res.fm.ecl` believe, but let's make that explicit.